lib/index.js

```diff
@@ -102,6 +103,11 @@ function mergeOptions(baseOptions, newOptions) {
 	const resolvedNewOptions = newOptions || {};
 	const headers = objectAssign({}, baseOptions.headers, resolvedNewOptions.headers);
 
+	const reqID = rTracer.id()?? null;
+	if (reqID != null) {
+		headers = objectAssign({}, headers, { 'request-id': reqID })
+	}
```

Comment:
```suggestion
	const reqID = rTracer.id() ?? null;
	if (reqID !== null)
		headers = objectAssign({}, headers, { 'request-id': reqID })
```

In JS, if the block statement is only 1 line, we remove the brackets.

Reply:
Additionally, because JS types are so... soft, we always check with triple equals:

`==` becomes `===`

and `!=` becomes `!==`

If you ever do want `!=`, then we normally add a comment saying it's intentional